packages/application/src/shell.ts

```diff
@@ -185,15 +196,25 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
   /**
    * Construct a new application shell.
    */
-  constructor() {
+  constructor(options?: ILabShell.IOptions) {
     super();
     this.addClass(APPLICATION_SHELL_CLASS);
     this.id = 'main';
 
+    const trans = ((options && options.translator) || nullTranslator).load(
+      'jupyterlab'
+    );
     const headerPanel = (this._headerPanel = new BoxPanel());
     const menuHandler = (this._menuHandler = new Private.PanelHandler());
+    menuHandler.panel.node.setAttribute('role', 'navigation');
+    menuHandler.panel.node.setAttribute(
+      'aria-label',
+      trans.__('main navigation')
```

Comment:
```suggestion
      trans.__('main')
```
My bad in the ticket I sent you. The word navigation is repeated twice which is just annoying. If screenreaders go to that element by jumping to it, viewing next element, or viewing all navigations, "main" is all that is needed.
![navigation of header nav labeled twice](https://user-images.githubusercontent.com/51969207/104762854-0abc6700-5733-11eb-9a60-f1d9d746fcaf.PNG)